From ec217570e20d4702be2830235bad56184d47b1d2 Mon Sep 17 00:00:00 2001
From: Greg Hudson <ghudson@mit.edu>
Date: Mon, 11 Mar 2013 13:48:13 -0400
Subject: [PATCH] Remove stray include in localauth_plugin.h

This unnecessary include was causing build failures on some systems by
making libkrb5 sources depend on gssapi.h.
---
 src/include/krb5/localauth_plugin.h |   1 -
 src/lib/krb5/os/deps                | 110 ++++++++++++------------------------
 src/plugins/localauth/test/deps     |  21 ++-----
 3 files changed, 41 insertions(+), 91 deletions(-)

diff --git a/src/include/krb5/localauth_plugin.h b/src/include/krb5/localauth_plugin.h
index 1c6165b..df7574e 100644
--- a/src/include/krb5/localauth_plugin.h
+++ b/src/include/krb5/localauth_plugin.h
@@ -62,7 +62,6 @@
 
 #include <krb5/krb5.h>
 #include <krb5/plugin.h>
-#include <kadm5/admin.h>
 
 /* An abstract type for localauth module data. */
 typedef struct krb5_localauth_moddata_st *krb5_localauth_moddata;
